webservices/partition/sched_a.py

```diff
@@ -57,7 +57,7 @@ def index_factory(cls, child):
             sa.Index(None, c.rpt_yr),
             sa.Index(None, c.entity_tp),
             sa.Index(None, c.image_num),
-            sa.Index(None, c.sched_a_sk),
+            sa.Index(None, c.sub_id),
```

Comment:
D'oh!  Apparently I missed that one, sorry.  We could also reference it like `c[cls.primary]` like I had elsewhere so we only ever change the primary key label in one spot.